frontend/src/Types.elm

```diff
@@ -36,14 +42,15 @@ type Msg
     | AttributionMsg AttributionMessage
     | ProfilingMsg ProfilingMessage
     | UrlChange Navigation.Location
+    | AddFile ( String, File )
 
 
 type AttributionMessage
     = SetLanguage Language
-    | ToggleInputMode Author
-    | SetText Author String
+    | SetFeatureCombo FeatureCombo
```

Comment:
It is a set of parameters (features in machine learning speak) for the backend. We choose a model based on this setting. Will add comments